### PR TITLE
[Rector] Apply Rector to app/Views

### PR DESCRIPTION
**Pull request by samsonasik**
**Description**

Apply Rector run into `app/Views` as well.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: app/Views/errors/html/error_exception.php [hunk 1]
```diff
@@ -289,9 +291,7 @@
                                 continue;
                             }
 
-                            if (! is_array($value)) {
-                                $value = [$value];
-                            } ?>
+                            $value = [$value]; ?>
```
Comment (samsonasik) [inline app/Views/errors/html/error_exception.php:294]: this change due to `$headers` is from `headers()` method which got `array<string, Header>` docblock return, so it always a `Header` detected, so to loop, it wrapped with `[]`, but probably incorrect docblock, it needs to be verified.

@MGatner  @kenjis @paulbalandan could you verify ? Thank you.

Comment (kenjis) [reply app/Views/errors/html/error_exception.php:294]: `$request->headers()` returns `array<string, Header>`.
https://github.com/codeigniter4/CodeIgniter4/blob/29c31173b6ecdd3be75ea69b3f5eb876e7b5dcbe/system/HTTP/MessageTrait.php#L116
https://github.com/codeigniter4/CodeIgniter4/blob/29c31173b6ecdd3be75ea69b3f5eb876e7b5dcbe/system/HTTP/MessageTrait.php#L154

So `$value` is never an array.
I sent a PR to refactor: #7171

File: app/Views/errors/html/error_exception.php [hunk 2]
```diff
@@ -1,4 +1,6 @@
-<?php $error_id = uniqid('error', true); ?>
+<?php use Config\Services;
```
Comment (MGatner) [inline app/Views/errors/html/error_exception.php:1]:
```suggestion
<?php
use Config\Services;
```

File: app/Views/errors/html/error_exception.php [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
+use CodeIgniter\CodeIgniter;
+$errorId = uniqid('error', true); ?>
```
Comment (MGatner) [inline app/Views/errors/html/error_exception.php:3]:
```suggestion
$errorId = uniqid('error', true);
?>
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Looks good! Can we make the imports more like our newline formatting on PHP files? I don't think Rector will care.

File: app/Views/errors/html/error_exception.php (hunk 2 above)
Comment (samsonasik) [reply app/Views/errors/html/error_exception.php]: updated https://github.com/codeigniter4/CodeIgniter4/pull/7169/commits/ba96124a1ef53714f6512b6e0ffa2b207f57de85

File: app/Views/errors/html/error_exception.php (hunk 3 above)
Comment (samsonasik) [reply app/Views/errors/html/error_exception.php]: updated https://github.com/codeigniter4/CodeIgniter4/pull/7169/commits/ba96124a1ef53714f6512b6e0ffa2b207f57de85

**Comment by samsonasik** [comment]
Thank you @kenjis @MGatner for the review.

**Pull request merged**